deploy/01_Deploy_PriceConsumerV3.js

```diff
@@ -7,26 +7,23 @@ module.exports = async ({
 }) => {
     const { deploy, log } = deployments
     const { deployer } = await getNamedAccounts()
-    let chainId = await getChainId()
+    const chainId = await getChainId()
     let ethUsdPriceFeedAddress
     if (chainId == 31337) {
-        MockV3Aggregator = await deployments.get('MockV3Aggregator')
+        const MockV3Aggregator = await deployments.get('EthUsdAggregator')
```

Comment:
Is this confusing?

Reply:
We should actually rename `const MockV3Aggregator` to `const EthUsdAggregator`. I just missed this and will update.